ovos_utils/ovos_service_api.py

```diff
@@ -178,9 +178,8 @@ def search_movie(self, query):
 
 
 class OvosGeolocate:
+    @deprecated("This reference is deprecated. Import from ovos-backend-client")
```

Comment:
can these point to the appropriate class under https://github.com/OpenVoiceOS/ovos-backend-client/blob/dev/ovos_backend_client/api.py